### PR TITLE
Fixes landice mesh tool interpolate_to_mpasli_grid.py when source and destination grids are both mpas format.

### PR DESCRIPTION
**Pull request by trhille**
Used input file for vertical dimensions, when should have used output file.

### CODE REVIEW

**Comment by xylar** [comment]
@trhille, it would be helpful for other users of MPAS-Tools if the name of the PR had some information about which component and/or tool is being modified.  But that way folks know whether it's relevant to them or not.  Easy enough to figure out once we go to the PR, but we're lazy ;-)

**Comment by matthewhoffman** [comment]
@xylar , you can blame for poor guidance on that.  That's a good point.  (@trhille , you can go in and edit the existing title from this page.)

**Comment by trhille** [comment]
@xylar @matthewhoffman Thanks. Should be better now.

**Comment by xylar** [comment]
Nice!  And no blame, just figured it would be helpful to know for future PRs, too.

File: landice/mesh_tools_li/interpolate_to_mpasli_grid.py [hunk 1]
```diff
@@ -478,7 +478,7 @@ def vertical_interp_MPAS_grid(mpas_grid_input_layers, input_layers):
 elif filetype == 'mpas':
     # Get the MPAS vertical dimensions if they exist
     try:
-      nVertLevels = len(inputFile.dimensions['nVertLevels'])
+      input_nVertLevels = len(inputFile.dimensions['nVertLevels'])
```
Comment (matthewhoffman) [inline landice/mesh_tools_li/interpolate_to_mpasli_grid.py]: @trhille , looking at this closer, this dimension variable is not actually used anywhere in the script, so a cleaner solution would be to simply delete this entire try/except clause (i.e. I never should have added it in the first place).  (The interpolation part figures out the vertical information for the source file around lines 280-291.)  You can delete these lines in a new commit and just push it to your branch, and the PR will update automatically.

**Comment by trhille** [comment]
@matthewhoffman Thanks. Done.

File: landice/mesh_tools_li/interpolate_to_mpasli_grid.py [hunk 2]
```diff
@@ -366,7 +366,7 @@ def interpolate_field_with_layers(MPASfieldName):
 def vertical_interp_MPAS_grid(mpas_grid_input_layers, input_layers):
     destinationField = np.zeros((nCells, nVertLevels))
     for i in range(nCells):
-        destinationField[i,:] = np.interp(mpasLayerCenters, input_layers, mpas_grid_input_layers[:,i])
+        destinationField[i,:] = np.interp(mpasLayerCenters,input_layers, mpas_grid_input_layers[:,i])
```
Comment (xylar) [inline landice/mesh_tools_li/interpolate_to_mpasli_grid.py]: I'm guessing this change was inadvertent?

Comment (matthewhoffman) [reply landice/mesh_tools_li/interpolate_to_mpasli_grid.py]: Yeah, and I was ignoring it in the attempt to keep things simple for an intro to PRing.

**Pull request merged**